src/main/flight/pid.h

```diff
@@ -73,8 +73,9 @@ typedef struct pid8_s {
 typedef struct pidProfile_s {
     pid8_t  pid[PID_ITEM_COUNT];
 
-    uint16_t yaw_lpf_hz;                    // Additional yaw filter when yaw axis too noisy
-    uint16_t dterm_lpf_hz;                  // Delta Filter in hz
+    uint16_t yaw_lowpass_hz;                    // Additional yaw filter when yaw axis too noisy
```

Comment:
PG broken

Reply:
PG?

Reply:
Parameter Group. Here: https://github.com/betaflight/betaflight/pull/5458/files/b849b242c16d42df9873baa43da22e65ba5164eb#diff-c5f9576172e273033324ba218524cc97R106

(looks correct though)

Reply:
@ctzsnooze: The structs used for parameters in parameter groups are also used to persist the parameter groups to EEPROM, and will be used to transfer the data over MSP in the future. As such some special constraints apply:
- new parameters can only be added at the end of the struct. This will retain compatibility with the current version of the parameter group;
- if more change than adding new parameters is needed (i.e. changing the type of parameters, removing parameters), this can be done, but the version number as defined in `PG_REGISTER_` needs to be incremented to indicate that the resulting parameter group is a new version, and incompatible with the previous version.

Reply:
Thanks for being patient with me.  

If I understand properly, the problem was in adding 'uint16_t dte.rm_lowpass2_hz;' up high in the struct for pidProfile_s, rather than putting it at the end, i.e. after 'uint16_t itermLimit;', without making a change in PG-REGISTER

Would it be sufficient for now just to move it to the end?  It might be best if I don't go out of my depth in making changes to PG-REGISTER.

Reply:
No problem, always happy to help people who want to help us to improve Betaflight.

And yes, adding the new parameter at the end of the struct is all that is needed to fix this. The renamed parameters don't matter, since the names are not used (for now).

Reply:
Thanks!  Done.